src/region.js

```diff
@@ -72,6 +72,10 @@ const Region = MarionetteObject.extend({
 
     this._attachView(view, options);
 
+    if (this._isReplaced) {
+      view.on('before:destroy', this._empty, this);
```

Comment:
hmm... should https://github.com/marionettejs/backbone.marionette/pull/3262/files#diff-b8da5292cb07ec757ac64e78ff5af5c1R67
be updated to 'before:destroy'?